core/src/test/java/com/linecorp/armeria/common/stream/StreamMessageTest.java

```diff
@@ -278,6 +286,27 @@ void abortedStream() {
                     .verify();
     }
 
+    @Test
+    void writeToFile(@TempDir Path tempDir) throws IOException {
+        final ByteBuf[] bufs = new ByteBuf[10];
+        for (int i = 0; i < 10; i++) {
+            bufs[i] = Unpooled.wrappedBuffer(Integer.toString(i).getBytes());
+        }
+        final byte[] expected = Arrays.stream(bufs)
+                .map(ByteBuf::array)
+                .reduce(Bytes::concat).get();
+
+        final StreamMessage<ByteBuf> publisher = StreamMessage.of(bufs);
+        final Path destination = tempDir.resolve("foo.bin");
+        publisher.writeTo(x -> HttpData.wrap(x), destination).join();
+        final byte[] bytes = Files.readAllBytes(destination);
+
+        assertThat(bytes).contains(expected);
```

Comment:
No, there's no reason. I just referred to `StreamMessagesTest`. 
I think it would be better to apply strict equality too. 😊

Would it be good to apply it to `StreamMessagesTest` too?